pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -664,7 +667,7 @@ func (k *Kubernetes) Deploy(komposeObject kobject.KomposeObject, opt kobject.Con
 			if err != nil {
 				return err
 			}
-			log.Infof("Successfully created PersistentVolumeClaim: %s", t.Name)
+			log.Infof("Successfully created PersistentVolumeClaim: %s of size %s. PVC's will not work unless a PersistentVolume has been created.", t.Name, PVCRequestSize)
```

Comment:
Cluster can be also setup with dynamic provisioning. Even new minikube is setup like this by default.
How about this:
`Successfully created PersistentVolumeClaim: %s of size %s. For PVC to work your cluster has to be setup with Dynamic provisioning of storage or PersistentVolume has to be created.`

Or something in those lines that mentions dynamic provisioning. I bet there is better sentence to explain it than what I've put together ;-)